test/dotnet-msbuild.Tests/GivenDotnetMSBuildBuildsProjects.cs

```diff
@@ -77,6 +86,37 @@ public void When_help_is_invoked_Then_MSBuild_extra_options_text_is_included_in_
         }
 
         [Fact]
+        public void WhenRestoreSourcesStartsWithUnixPathThenHttpsSourceIsParsedCorrectly()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
I feel like what we should do is create a NonWindowsFact, a counterpart of our WindowsOnlyFactAttribute.